tensortrade/oms/wallets/wallet.py

```diff
@@ -341,13 +343,14 @@ def transfer(source: 'Wallet',
 
         q = quantity.size
         c = commission.size
+        qc = (q + c) if c > 0.0 else q
         cv = converted.size
         p = exchange_pair.inverse_price if pair == exchange_pair.pair else exchange_pair.price
 
         source_quantization = Decimal(10) ** -source.instrument.precision
         target_quantization = Decimal(10) ** -target.instrument.precision
 
-        lhs = Decimal((lsb1 - lsb2) - (q + c)).quantize(source_quantization)
+        lhs = Decimal(lsb1 - lsb2 - qc).quantize(source_quantization)
```

Comment:
Maybe adding some comments would be good to explain a bit what's happening.

Reply:
Quantity + commission are precalculated so nothing special changed here.